Documentation/project-docs/platform-names.md

```diff
@@ -4,6 +4,24 @@ Over the years, we've accumulated quite a few names, platforms, and components t
 
 This will also list some aliases. As you'll see the aliases aren't always correct -- they are merely listed to help you find the better and less confusing terminology.
 
+In this document, the following terms are used:
+
+* **IL**. Intermediate language. IL is a stack-based instruction set and is what higher level languages, such as C#, compile down to.
```

Comment:
Is CIL (Common Intermediate Language) still called MSIL? I thought the latter had been deprecated. Both CIL and MSIL should probably be mentioned.

I think "stack-based" without explanation is enough for someone to understand, not care about for now, or google, depending on both knowledge and inclination.